src_c/polygon.c

```diff
@@ -358,7 +467,68 @@ pg_polygon_get_vertices(pgPolygonObject *self, void *closure)
     return _pg_polygon_vertices_aslist(&self->polygon);
 }
 
+static PyObject *
+pg_polygon_get_center_x(pgPolygonObject *self, void *closure)
+{
+    return PyFloat_FromDouble(self->polygon.c_x);
+}
+
+static int
+pg_polygon_set_center_x(pgPolygonObject *self, PyObject *value, void *closure)
+{
+    double val;
+    DEL_ATTR_NOT_SUPPORTED_CHECK_NO_NAME(value);
+    if (!pg_DoubleFromObj(value, &val)) {
+        PyErr_SetString(PyExc_TypeError, "Expected a number");
+        return -1;
+    }
+    self->polygon.c_x = val;
+    return 0;
+}
+
+static PyObject *
+pg_polygon_get_center_y(pgPolygonObject *self, void *closure)
+{
+    return PyFloat_FromDouble((int)self->polygon.c_y);
+}
+
+static int
+pg_polygon_set_center_y(pgPolygonObject *self, PyObject *value, void *closure)
+{
+    double val;
+    DEL_ATTR_NOT_SUPPORTED_CHECK_NO_NAME(value);
+    if (!pg_DoubleFromObj(value, &val)) {
+        PyErr_SetString(PyExc_TypeError, "Expected a number");
+        return -1;
+    }
+    self->polygon.c_y = val;
+    return 0;
+}
+
+static PyObject *
+pg_polygon_get_center(pgPolygonObject *self, void *closure)
+{
+    return Py_BuildValue("(dd)", self->polygon.c_x, self->polygon.c_y);
```

Comment:
You should use the `pg_tuple_from_values_double` function instead of `Py_BuildValue`, it's 40% faster.
```suggestion
    return pg_tuple_from_values_double(self->polygon.c_x, self->polygon.c_y);
```

Reply:
Done! 

FYI: I will write "Done!" for the ones I finish and then commit changes.